transcode/transcode.go

```diff
@@ -59,10 +61,6 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 	if err != nil {
 		return outputs, segmentsCount, err
 	}
-	fmp4TargetURL, err := getFmp4TargetURL(transcodeRequest)
```

Comment:
I think this should still be here to validate that the url was set correctly and then in line 182, use `fmp4TargetURL` instead of `transcodeRequest.FragMp4TargetUrl`.

Reply:
It should be ok I think, we're calling `getMp4OutputType` instead which does the same stuff as `getFmp4TargetURL` did, it checks if the string is empty and if not parses it to a url